src/components/orgs/UserManagement.vue

```diff
@@ -1,27 +1,28 @@
 <template>
   <div>
     <div class="group">
-      <org-permission-select
-        v-model="role"
-        :label="labelRole"
-        class="weni-org-permissions__permission-select"
-      />
-
       <unnnic-input
         class="flex-1"
         v-model="userSearch"
         :type="userError ? 'error' : 'normal'"
         :message="userError"
         :label="labelEmail"
         :placeholder="$t('orgs.create.user_search_description')"
-        icon-right="keyboard-return-1"
-        :tooltip-icon-right="$t('orgs.create.press_enter_to_add')"
-        :tooltip-side-icon-right="tooltipSideIconRight"
-        :tooltip-force-open-icon-right="forceTooltipPressEnterOpen"
         @keyup.enter="onSubmit"
         @input="userError = null"
         :disabled="loadingAddingUser"
       />
+
+      <div class="group__right">
+        <unnnic-button 
+          @click="onSubmit" 
+          type="secondary" 
+          :disabled="!!!userSearch || loadingAddingUser"
```

Comment:
Perhaps this !!!userSearch could be change to !userSearch, that will make same behavior

Reply:
Okay!! Fixed in next commit!